platform/openide.util/test/unit/src/org/openide/util/TaskTest.java

```diff
@@ -147,7 +310,7 @@ public synchronized void run () {
             }
         }
     }
-    
+    @Test
     /*
      * see issue #130265
      */
```

Comment:
nitpick: move below comment

Reply:
done in commit https://github.com/apache/netbeans/pull/4112/commits/3d0f3c0bd2831bcb76cb6001d4c5e25a17b8917d